rosdep/python.yaml

```diff
@@ -4859,6 +4859,13 @@ python-sqlalchemy:
 python-sqlite:
   debian: [python-sqlite]
   ubuntu: [python-sqlite]
+python-staticmap-pip:
+  debian:
+    pip:
+      packages: [staticmap]
+  ubuntu:
+    pip:
+      packages: [staticmap]
```

Comment:
@sktometometo it is somewhat unclear which Python versions does `staticmap` support.

Isn't there a better supported alternative we can use instead?

Reply:
staticmap supports both python 2 and python 3. 
 In this case, pip package names are identical for both python version.

https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1 shows an alternative like.

```yaml
python-foobar-pip: &migrate_eol_2025_04_30_python3_foobar_pip # Anchor
  ubuntu:
    pip:
      packages: [foobar]
python3-foobar-pip: *migrate_eol_2025_04_30_python3_foobar_pip # Alias
```

Should I use this kind of format?

Reply:
So I'd generally prefer if we only add in the `python3-*-pip` key.  Do you need the Python2 version of the key?